src/util/scroll.js

```diff
@@ -160,6 +160,15 @@ function scrollToPosition (shouldScroll, position) {
   }
 
   if (position) {
-    window.scrollTo(position.x, position.y)
+    // $flow-disable-line
+    if ('scrollBehavior' in document.documentElement.style) {
+      window.scrollTo({
+        left: position.x,
+        top: position.y,
+        behavior: shouldScroll.behaviour || 'auto'
```

Comment:
we can pass undefined, so we should be able to just do:

```suggestion
        behavior: shouldScroll.behaviour
```